tests/util/mod.rs

```diff
@@ -338,15 +339,17 @@ async fn setup_container(port: i32) -> Result<(), Box<dyn std::error::Error>> {
     io::stderr().write_all(&output.stderr).unwrap();
 
     // Wait dynamodb-local
-    let health_check_url = format!("http://localhost:{}", port);
-    let ddb = DynamoDbClient::new(Region::Custom {
-        name: "local".to_owned(),
-        endpoint: health_check_url,
-    });
+    // https://docs.aws.amazon.com/sdk-for-rust/latest/dg/dynamodb-local.html
+    let config = aws_sdk_dynamodb::config::Builder::from(
+        &SdkConfig::builder().region(Region::new("local")).build(),
```

Comment:
Need to set `behavior_version`.
```suggestion
        &SdkConfig::builder()
            .region(Region::new("local"))
            .behavior_version(BehaviorVersion::v2024_03_28())
            .build(),
```